src/csharp/Utils.cs

```diff
@@ -36,7 +39,10 @@ protected virtual void Dispose(bool disposing)
         }
     }
 
-    public class Utils
+    /// <summary>
+    /// Wraps some utility functions that are available globally
+    /// <summary>
+    internal class Utils
```

Comment:
In Java there's a `GenAI` class that handles the global configurations:

https://github.com/microsoft/onnxruntime-genai/blob/d12927416fa791802ce73fa795129f1e14f3d42a/src/java/src/main/java/ai/onnxruntime/genai/GenAI.java#L19

Should we do the same in C#?